doc/src/manual/performance-tips.md

```diff
@@ -1173,44 +1117,45 @@ The common idiom of using 1:n to index into an AbstractArray is not safe if the
 and may cause a segmentation fault if bounds checking is turned off. Use `linearindices(x)` or `eachindex(x)`
 instead (see also [offset-arrays](https://docs.julialang.org/en/latest/devdocs/offset-arrays)).
 
-Note: While `@simd` needs to be placed directly in front of a loop, both `@inbounds` and `@fastmath`
-can be applied to several statements at once, e.g. using `begin` ... `end`, or even to a whole
-function.
+!!!note
+    While `@simd` needs to be placed directly in front of a loop, both `@inbounds` and `@fastmath`
+    can be applied to several statements at once, e.g. using `begin` ... `end`, or even to a whole
+    function.
 
 Here is an example with both `@inbounds` and `@simd` markup:
 
 ```julia
-function inner(x, y)
+@noinline function inner(x, y)
```

Comment:
Have to put `@noinline` here or this gets inlined and optimized away in `timeit` below. See https://github.com/JuliaLang/julia/issues/27029